src/components/CommentList/CommentList.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { CommentPropTypes } from '../PropTypes/CommentPropTypes';
+
+import './CommentList.scss';
+
+import { Comment } from '../Comment';
+
+export const CommentList = ({ comments }) => (
+  <div>
+    <h2 className="commentlist__title">Comments</h2>
+    <ul className="commentlist list-group">
+      {
+        comments.map(comment => (
+          <li
+            className="list-group-item
+              list-group-item-action
+              list-group-item-light"
```

Comment:
```suggestion
            className=`
              list-group-item
              list-group-item-action
              list-group-item-light
            `
```
по-моему у тебя тут должна быть ошибка, потому что нельзя переносить строку в `""` или `''`

Reply:
c бектиками все красное, не пропускает, линтер только с обычными дает сохранить"", перенос на новую строчку сделала с ними же, могу убрать один класс, тогда влезет в 1 строчку (в демке не будет выделения комментов)